qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -78,6 +78,7 @@
                     cpu_model_flags += ",+xsaveerptr,+clzero"
                     flags = "clzero xsaveerptr"
                 - virt-ssbd:
+                    check_models = "EPYC-Rome EPYC EPYC-IBPB"
```

Comment:
How about "supported_models = ..." ?

Reply:
> How about "supported_models = ..." ?

It shouldn't say that the other model doesn't support, this change is just for testing easily.

Reply:
@nanliu-r  IMO, exclude the model that do not need to test is more suitable than define the models that need to check.  Since we are not sure about the status with new cpu module.

Reply:
> @nanliu-r IMO, exclude the model that do not need to test is more suitable than define the models that need to check. Since we are not sure about the status with new cpu module.

ok, will update this, thanks.